### PR TITLE
Added MultipleValues Meter story

### PR DESCRIPTION
**Pull request by ericsoderberghp**
#### What does this PR do?

Added MultipleValues Meter story.
Note the update storybook config to allow a component to have a `stories` directory.

### CODE REVIEW

**Comment by oorestisime** [comment]
it may be only me but i find this example confusing.

If there are 70gb storage and 40 are used and 30 are available what is the grey value meaning?

File: .eslintrc [hunk 1]
```diff
@@ -43,7 +43,7 @@
   },
   "overrides": [
     {
-      "files": ["*.stories.js"],
+      "files": ["*.stories.js", "**/stories/**"],
```
Comment (ShimiSun) [inline .eslintrc]: NICE! we might want to move the Chart stories in the same way.

File: src/js/components/Meter/stories/MultipleValues.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box, Meter, Stack, Text } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+class MultipleValues extends Component {
+  state = { total: 70 };
```
Comment (ShimiSun) [inline src/js/components/Meter/stories/MultipleValues.js]: might make more sense to be ` state = { total: 100 }`. unless I'm missing something.

**Comment by ShimiSun** [comment]
Can you please move `src/js/components/Meter/meter.stories.js` to be under the new stories directory. this way we won't have stories in different places of the Meter hierarchy.

File: src/js/components/Meter/stories/MultipleValues.js (hunk 2 above)
Comment (ericsoderberghp) [reply src/js/components/Meter/stories/MultipleValues.js]: Good idea, changed.

File: .eslintrc (hunk 1 above)
Comment (ericsoderberghp) [reply .eslintrc]: At some point, yes.

**Comment by ericsoderberghp** [comment]
> Can you please move `src/js/components/Meter/meter.stories.js` to be under the new stories directory. this way we won't have stories in different places of the Meter hierarchy.

done

**Pull request merged**